sirenity/code/handler.py

```diff
@@ -0,0 +1,43 @@
+import ast
+import os
+
+allowed_modules = ["math"]
+
+
+class CodeHandler:
+    """A class used to handle code submissions from users"""
+
+    pass
```

Comment:
```suggestion
class CodeHandler:
    """A class used to handle code submissions from users"""
```
`pass` isn't required here